org-noter/org-noter
# Add support for semantic versioning

## Problem

We've been stuck on version 1.5.0 since last year.

## Solution

Use [svu](https://github.com/caarlos0/svu) to manage org-noter's semantic version along with a few Makefile changes.

## Steps to Test

1. Pull this branch
2. Install [svu](https://github.com/caarlos0/svu)
3. Run `make bump-patch` or `make bump-minor` to increment the appropriate version. 

- See that an appropriate tag is added (`git tag -l`)
- See that the exact same version string is applied to the `;;Version` header in `org-noter.el`.
